feder/main/templates/base.html

```diff
@@ -71,26 +71,25 @@ <h1 class="sr-only">{% trans 'Civil data mining' %}</h1>
                     </a>
                     <ul class="dropdown-menu">
                         <li>
-                            {# performant paginator does not support last #}
                             <a href="{% url 'cases:list' %}">
                                 <i class="fa fa-folder-open"></i>
                                 {% trans 'Cases' %}
                             </a>
                         </li>
                         <li>
-                            <a href="{% url 'monitorings:list' %}?page=last">
```

Comment:
Wouldn't it be better to leave displaying the last page for monitorings for convenience? We had this feature requested some time ago.